packages/odie-client/src/components/message/user-message.tsx

```diff
@@ -57,14 +51,7 @@ export const UserMessage = ( {
 		);
 		const isLastMessage = currentMessageIndex === chat.messages.length - 1;
 
-		return (
-			isLastMessage &&
-			( shouldUseHelpCenterExperience ? (
-				<GetSupport onClickAdditionalEvent={ handleContactSupportClick } />
-			) : (
-				extraContactOptions
-			) )
-		);
+		return isLastMessage && <GetSupport onClickAdditionalEvent={ handleContactSupportClick } />;
```

Comment:
I think is preferable to return NULL. Eg.
```suggestion
		return isLastMessage ? <GetSupport onClickAdditionalEvent={ handleContactSupportClick } /> : null;
```